apps/linear_algebra/Makefile

```diff
@@ -7,8 +7,8 @@ EMIT_OPTIONS = stmt,assembly
 
 EIGEN_INCLUDES ?= -I/usr/include/eigen3
 CBLAS_LIBS ?= -lblas
-ATLAS_FLAGS ?= -DUSE_ATLAS
-ATLAS_LIBS ?= -L/usr/lib/atlas-base -lcblas -latlas
+ATLAS_FLAGS ?= -DUSE_ATLAS -I/opt/ATLAS/include
+ATLAS_LIBS ?= -L/opt/ATLAS/lib -lptcblas -latlas
```

Comment:
Yeah, I was a bit unsure about checking these lines in. But installing it locally is not just to get the pthreads version, Rather the packaged versions are kind of useless, as the whole point of the library is that is tunes its performance for the specific machine during the build.

I can revert these lines of the Makefile for this PR though.